cmake/modules/SwiftSharedCMakeConfig.cmake

```diff
@@ -293,16 +289,10 @@ macro(swift_common_unified_build_config product)
     get_filename_component(CMARK_LIBRARY_DIR "${${product}_CMARK_LIBRARY_DIR}"
       ABSOLUTE)
 
-    set(CMARK_BUILD_INCLUDE_DIR "${PATH_TO_CMARK_BUILD}/src")
-    set(CMARK_MAIN_INCLUDE_DIR "${CMARK_MAIN_SRC_DIR}/src/include")
+    include_directories(${PATH_TO_CMARK_BUILD}/src)
```

Comment:
same, why not merge these into one `include_directories(${PATH_TO_CMARK_BUILD}/src ${CMARK_MAIN_SRC_DIR}/src/include)`?

Reply:
We could, and they would be semantically identical. I just split them up as I did the replacement one at a time, and I intend to just get rid of all of this in favour of imported targets for cmark globally.